### PR TITLE
New fork for distilbert

### PR DESCRIPTION
**Pull request by arunimaaaa**

### CODE REVIEW

File: notebooks/224-distilbert-Sequence-Classification/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+transformers
+onnx
```
[the remark this answers is not in the document]
Comment (arunimaaaa) [reply notebooks/224-distilbert-Sequence-Classification/requirements.txt:2]: Shall I remove the file and update readme?

File: notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification-OV2.0.ipynb [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e31a2644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.filterwarnings(\"ignore\")\n",
+    "core = ov.Core()\n",
+    "ir_model_xml = str((Path(MODEL_DIR)/checkpoint).with_suffix(\".xml\"))\n",
+    "compiled_model = core.compile_model(ir_model_xml, \"AUTO\")\n",
```
Comment (eaidova) [inline notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification-OV2.0.ipynb:114]: for AUTO device assignment no need specify device, this option selected by default

File: notebooks/224-distilbert-Sequence-Classification/Readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Sequence Classification With OpenVINO API 2.0
+This notebook performs sequence classification using OpenVINO API 2.0. You can alternatively replace the model with the other bert-based models for sequence classification.
```
Comment (eaidova) [inline notebooks/224-distilbert-Sequence-Classification/Readme.md:2]:
```suggestion
This notebook performs sequence classification using OpenVINO API 2.0. and `distilbert-base-uncased` model from HuggingFace transformers finetuned on [SST-2](https://huggingface.co/datasets/sst2) dataset. More information about this model can be found in [model card](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english). You can alternatively replace the model with the other bert-based models for sequence classification.

```

File: notebooks/224-distilbert-Sequence-Classification/Readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+![image](https://user-images.githubusercontent.com/95271966/203713415-669a755d-3243-4e66-b19b-cd17774a1a64.png)
+
+
+Output from a file would look something like this.
+![image](https://user-images.githubusercontent.com/95271966/203713154-b78b383e-ec42-4b3b-a142-47b00640bdea.png)
+
+
+## Notebook Contents
+We'll be using distilbert-base-uncased-finetuned-sst-2-english transformers based model from Huggingface. Details of the model can be found [here].(https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english)
```
Comment (eaidova) [inline notebooks/224-distilbert-Sequence-Classification/Readme.md:15]: I made suggestion to put it to description as it is general info about notebook. I think here should be answer what this notebook do?
Like:
 this notebook demostrates how to convert model to ONNX format, then OpenVINO Intermediate Representation, then run model inference on user specified input text and from file for performing sentiment analysis.

File: notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification-OV2.0.ipynb [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e31a2644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.filterwarnings(\"ignore\")\n",
+    "core = ov.Core()\n",
+    "ir_model_xml = str((Path(MODEL_DIR)/checkpoint).with_suffix(\".xml\"))\n",
```
[the remark this answers is not in the document]
Comment (arunimaaaa) [reply notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification-OV2.0.ipynb:113]: If I pass as a Path:
image

File: notebooks/224-distilbert-Sequence-Classification/requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (arunimaaaa) [reply notebooks/224-distilbert-Sequence-Classification/requirements.txt]: Done

File: README.md [hunk 6]
```diff
@@ -146,6 +146,7 @@ More amazing notebooks here!
...
 | [223-gpt2-text-prediction](notebooks/223-gpt2-text-prediction/)<br> | Use GPT-2 to perform text prediction on an input sequence | <img src=https://user-images.githubusercontent.com/91228207/185105225-0f996b0b-0a3b-4486-872d-364ac6fab68b.png  width=225> |
 | [224-3D-segmentation-point-clouds](notebooks/224-3D-segmentation-point-clouds/)<br> | Process point cloud data and run 3D Part Segmentation with OpenVINO | <img src = "https://user-images.githubusercontent.com/91237924/185752178-3882902c-907b-4614-b0e6-ea1de08bf3ef.png" width=225> |
+| [224-distilbert-Sequence-Classification](notebooks/224-distilbert-Sequence-Classification/)<br> | Sequence Classification With OpenVINO API 2.0 | <img src = "https://user-images.githubusercontent.com/95271966/203713415-669a755d-3243-4e66-b19b-cd17774a1a64.png" width=225> |
```
Comment (eaidova) [inline README.md:149]:
```suggestion
| [225-distilbert-Sequence-Classification](notebooks/225-distilbert-Sequence-Classification/)<br> | Sequence Classification With OpenVINO API 2.0 | <img src = "https://user-images.githubusercontent.com/95271966/203713415-669a755d-3243-4e66-b19b-cd17774a1a64.png" width=225> |
```

**Pull request closed without merging**